packages/media-server/srv/media-service.js

```diff
@@ -0,0 +1,83 @@
+module.exports = (srv) => {
+
+    var loki = require('lokijs');
+    var db = new loki('DB');
+    var mediaDB= db.addCollection("Media");
+    var Readable = require('stream').Readable;
+    var PassThroughStream = require('stream').PassThrough;
```

Comment:
better `const`, not `var` and `PassThrough` instead of `PassThroughStream`.
using { Readable, PassThrough } = require('stream')

Reply:
I have made the changes as suggested